pkg/framework/deployment.go

```diff
@@ -34,7 +40,12 @@ func GetDeployment(c client.Client, name, namespace string) (*kappsapi.Deploymen
 
 // DeleteDeployment deletes the specified deployment
 func DeleteDeployment(c client.Client, deployment *kappsapi.Deployment) error {
```

Comment:
why do we need to wrap this? now `DeleteDeployment` is unused. Can't we just have a single `waitForDerploymentDeletion`